src/store/modules/recent/actions.ts

```diff
@@ -0,0 +1,51 @@
+import { SpaceConnector } from '@spaceone/console-core-lib/space-connector';
+import ErrorHandler from '@/common/composables/error/errorHandler';
+import { RECENT_TYPE, RecentConfig, RecentState } from '@/store/modules/recent/type';
+import { Action } from 'vuex';
+
+
+const createRecent = async (itemType: RECENT_TYPE, itemId: string) => {
+    try {
+        await SpaceConnector.client.addOns.recent.create({
+            type: itemType,
+            id: itemId,
+        });
+    } catch (e) {
+        ErrorHandler.handleError(e);
+    }
+};
+
+export const addItem: Action<RecentState, any> = async ({ commit }, recent: RecentConfig): Promise<void> => {
+    const itemType = recent.itemType;
+    const itemId = recent.itemId;
+    await createRecent(itemType, itemId);
+    if (itemType === RECENT_TYPE.PROJECT) {
+        commit('addProjectItem', recent);
+    } else if (itemType === RECENT_TYPE.PROJECT_GROUP) {
+        commit('addProjectGroupItem', recent);
+    } else if (itemType === RECENT_TYPE.CLOUD_SERVICE) {
+        commit('addCloudServiceTypeItem', recent);
+    } else if (itemType === RECENT_TYPE.MENU) {
+        commit('addMenuItem', recent);
+    }
```

Comment:
간결성과 명확성을 부여하기 위해 literal object 방식도 좋을 것 같습니다.
근데 이것도 좋아요 !!! 
괜히 한번 남겨봅니다.

```suggestion
const recentCommitsByItemType = {
  [RECENT_TYPE.PROJECT]: 'addProjectItem',
  [RECENT_TYPE.PROJECT_GROUP]: 'addProjectGroupItem',
  [RECENT_TYPE.CLOUD_SERVICE]: 'addCloudServiceTypeItem',
  [RECENT_TYPE.PROJECT]: 'addMenuItem',
}
commit(recentCommitsByItemType[itemType], recent);
```

Reply:
헉 넘 좋아요!!!!!!
감사합니다!